pyproject.toml

```diff
@@ -9,8 +9,8 @@ path = 'pydantic_extra_types/__init__.py'
 name = 'pydantic-extra-types'
 description = 'Extra Pydantic types.'
 authors = [
-    {name = 'Samuel Colvin', email = 's@muelcolvin.com'},
-    {name = 'Yasser Tahiri', email = 'hello@yezz.me'},
+    { name = 'Samuel Colvin', email = 's@muelcolvin.com' },
+    { name = 'Yasser Tahiri', email = 'hello@yezz.me' },
```

Comment:
There's clearly a formatting difference between the existing and the dev container. This should be rectified.